tree/ntuple/v7/src/RField.cxx

```diff
@@ -1394,8 +1394,12 @@ void ROOT::Experimental::RField<float>::SetHalfPrecision()
 const ROOT::Experimental::RFieldBase::RColumnRepresentations &
 ROOT::Experimental::RField<double>::GetColumnRepresentations() const
 {
-   static RColumnRepresentations representations(
-      {{EColumnType::kSplitReal64}, {EColumnType::kReal64}, {EColumnType::kSplitReal32}, {EColumnType::kReal32}}, {});
+   static RColumnRepresentations representations({{EColumnType::kSplitReal64},
+                                                  {EColumnType::kReal64},
+                                                  {EColumnType::kSplitReal32},
+                                                  {EColumnType::kReal32},
+                                                  {EColumnType::kReal16}},
```

Comment:
If I'm pedantic, this should probably be in the first commit to actually enable the `double` -> `kReal16` conversion...